### ISSUE TITLE
Camera parameters

### ISSUE THREAD

**Issue by anilesec** [NVlabs/dex-ycb-toolkit#1]
Thanks for the really cool work and the dataset.
I am just wondering, does that released dataset contains camera parameters(extrinsic and intrinsic)?

Thank you!

**Comment by ychao-nvidia** [comment]
Yes, it does. The intrinsics and extrinsics are provided in the dataset (see the [project site](https://dex-ycb.github.io) for download).

**Comment by anilesec** [comment]
I checked the calibration folder,  but there are 10 extrinsic.yml files. It is supposed to be 8m right? because we only have 8 fixed cameras. Additionally each extrinsic.yml file has more than one set of extrinsics. I do not understand how to decode this yml file. Could you please explain how to understand the info in extrinsic.yml file? 

Thank you!

**Comment by ychao-nvidia** [comment]
The 10 extrinsic.yml files correspond to the 10 subjects. We captured each subject in different dates so we re-calibrated the cameras before capturing a subject. If you open `meta.yml` in a sequence's folder, you'll see extrinsics ID for that sequence, e.g., `20200702_151821` for the sequence `20200709-subject-01/20200709_141754`.

Each extrinsics.yml file contains the extrinsics for all the 8 cameras. For example, if you open `extrinsics_20200702_151821/extrinsics.yml`, you should see:

```
extrinsics:
  '836212060125': !!python/tuple
  - -0.894641101360321
  - -0.04048347473144531
  - -0.44494757056236267
  - 0.6362327337265015
  - -0.31882184743881226
  - 0.7555275559425354
  - 0.5723029971122742
  - -0.6008626222610474
  - 0.3130013048648834
  - 0.6538649201393127
  - -0.6888331174850464
  - 1.3911463022232056
  '839512060362': !!python/tuple
  - -0.22983123362064362
  - 0.262200266122818
  - 0.9372453093528748
  - -0.6329705715179443
  - 0.7641939520835876
  - 0.6449487209320068
  - 0.006967182271182537
  - -0.07978752255439758
  - -0.6026484370231628
  - 0.71783846616745
  - -0.34860125184059143
  - 1.0656930208206177
...
```

`836212060125` is the ID of one camera and the following numbers represent the 3x4 transformation matrix (for rotation and translation).

To understand how we consume these extrinsics files, you can look at the example of the [interactive 3D viewer](https://github.com/NVlabs/dex-ycb-toolkit#interactive-3d-viewer). The extrinsics are loaded in [this line](https://github.com/NVlabs/dex-ycb-toolkit/blob/bcbed06f7d0fc523e9ef8b496abf495e4bff23ee/dex_ycb_toolkit/sequence_loader.py#L87).
